gossip/src/cluster_info.rs

```diff
@@ -1668,7 +1668,7 @@ impl ClusterInfo {
             self.gossip
                 .purge(&self_pubkey, thread_pool, timestamp(), &timeouts)
         };
-        inc_new_counter_info!("cluster_info-purge-count", num_purged);
+        inc_new_counter_debug!("cluster_info-purge-count", num_purged);
```

Comment:
@behzadnouri any objections to this? Is this still a useful metric?

Reply:
https://github.com/solana-labs/solana/pull/24451